framework/src/media/utils/MediaUtils.cpp

```diff
@@ -462,5 +462,66 @@ bool header_parsing(unsigned char *buffer, unsigned int bufferSize, audio_type_t
 	}
 	return true;
 }
+
+void write_little_endian(FILE *fp, unsigned int num, unsigned int byte)
+{
+	unsigned char buf[2];
```

Comment:
you don't need buf[2];
you can using `unsinged char buf` only.
this function buf[1] is not using.